src/components/Calculator.js

```diff
@@ -1,57 +1,105 @@
 /* eslint-disable react/prefer-stateless-function */
 /* eslint-disable react/no-array-index-key */
```

Comment:
- [ ] You have **disabled linter check** on your code. Please **enable linter check**  on your code to ensure that your code is quality and robust.